etc/webpack.base.js

```diff
@@ -5,7 +5,7 @@ var config = require('../lib/config');
 
 module.exports = new WebpackConfig().merge({
   entry: [
-    'normalize.css',
+    ...config.cssAdditionalEntry,
```

Comment:
I think this should be ES5 - I think we should stick to vanilla ES5 for core (for now)